### PR TITLE
Improve test coverage i89

### PR DESCRIPTION
**Pull request by hadialqattan**
Please don't merge before answering my question on issue #89

### CODE REVIEW

File: tests/test_metadata.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from semantic_version import Version
+
+from unimport.constants import DESCRIPTION, VERSION
+
+
+class TestMetadata(unittest.TestCase):
+    def test_description(self):
+        self.assertEqual(type(DESCRIPTION), str)
```
Comment (hakancelik96) [inline tests/test_metadata.py:10]:
```suggestion
        self.assertIsInstance(DESCRIPTION, str)
```

File: tests/test_metadata.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        self.assertGreater(len(DESCRIPTION), 0, "Too short description.")
+
+    def test_version(self):
+        try:
+            valid = bool(Version(VERSION))
+        except ValueError:
+            valid = False
+        # It follows strictly the 2.0.0 version of the SemVer scheme.
+        # For more information: https://semver.org/spec/v2.0.0.html
+        self.assertTrue(valid, "Invalid semantic-version.")
```
Comment (hakancelik96) [inline tests/test_metadata.py:20]:
```suggestion
        # It follows strictly the 2.0.0 version of the SemVer scheme.
        # For more information: https://semver.org/spec/v2.0.0.html
        self.assertTrue(Version(VERSION), "Invalid semantic-version.")
```

**Review by hakancelik96 — CHANGES_REQUESTED** [review]

File: unimport/refactor.py [hunk 3]
```diff
@@ -132,7 +132,6 @@ def get_star_imp():
             imp = get_star_imp()
             if imp:
                 return self.leave_StarImport(original_node, updated_node, imp)
```
Comment (hakancelik96) [inline unimport/refactor.py:134]:
```suggestion
            return self.leave_StarImport(original_node, updated_node, get_star_imp())
```

**Review by hakancelik96 — CHANGES_REQUESTED** [review]

File: unimport/refactor.py (hunk 3 above)
Comment (hakancelik96) [reply unimport/refactor.py:134]: The bool value of imp is always True.

File: tests/test_metadata.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from semantic_version import Version
```
Comment (hakancelik96) [inline tests/test_metadata.py:3]:
```suggestion
import semantic_version
```

File: tests/test_metadata.py (hunk 2 above)
Comment (hakancelik96) [inline tests/test_metadata.py:20]:
```suggestion
        # It follows strictly the 2.0.0 version of the SemVer scheme.
        # For more information: https://semver.org/spec/v2.0.0.html
        self.assertIsInstance(VERSION, str)
        self.assertTrue(semantic_version.validate(VERSION), "Invalid semantic-version.")
```

**Review by hakancelik96 — CHANGES_REQUESTED** [review]

File: tests/test_color.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+        reset = "\033[0m"
+        styled_str = Color("Unimport")
+        self.assertGreater(len(COLORS), 0, "No-colors")
+        color = list(COLORS.keys())[0]
+        value = COLORS[color]
+        template = styled_str.template(color)
+        self.assertTrue(template.startswith(value))
+        self.assertTrue(template.endswith(reset))
+        self.assertTrue(getattr(styled_str, color).startswith(value))
+        self.assertTrue(getattr(styled_str, color).endswith(reset))
```
[the remark this answers is not in the document]
Comment (hadialqattan) [reply tests/test_color.py]: Your approach is extraordinary, love it!

File: unimport/refactor.py (hunk 3 above)
Comment (hadialqattan) [reply unimport/refactor.py:134]: Yeah, I'm sorry about that!

**Review by hakancelik96 — APPROVED** [review]

**Comment by hakancelik96** [comment]
Thank you for your great contribution.

**Comment by hadialqattan** [comment]
> Thank you for your great contribution.

That's my pleasure!

**Pull request merged**